pom.xml

```diff
@@ -49,10 +49,11 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <java.version>11</java.version>
+        <java.version>11</java.version>
         <kotlin.version>1.9.20</kotlin.version>
         <kotlin.coroutines.version>1.7.3</kotlin.coroutines.version>
         <neo4j.version>4.4.27</neo4j.version>
-        <kafka.version>2.6.3</kafka.version>
+        <kafka.version>3.7.0</kafka.version>
```

Comment:
Same as in #619.

Reply:
3.4 is the lowest version containing the fix. As we discussed used it.